### DISCUSSION TITLE
Getting the  Value of a function Parameter

### DISCUSSION THREAD

**Discussion by martytx56** [Vector35/binaryninja-api#2259] [API] [answered]
Consider a function foo with 1 argument
If a function is called like this call foo(1), I can determine in a Python console script what the value of the argument passed to it.

But what about this case:

junk = 1
call foo(junk)

All I get is the name of the variable (junk) but I want it's value.  I can use the SSA form version since the name junk will be unique.

**Comment by plafosse** [comment, marked as answer]
You can just use the `.value` or `.possible_values` on the variable object.
